tests/Fixtures/inc/functions/rocketGetDirectoryPHPFilesArray.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Data Provider for _rocketGetDirectoryPHPFilesArray() tests.
+ *
+ * @author  Caspar Green
+ * @since   ver 3.6.1
+ */
+
+return [
+	'vfs_dir' => 'wp-content/wp-rocket-config/',
+
```

Comment:
Let's create a `structure` for the different scenarios. That will simplify the test method.

Huh? Take all of the `contents` part of the test data below and move it into here:

```php
'structure' => [
	'wp-content' => [
		'wp-rocket-config' => [
				'.', => '',
				'..' => '',
				'example.org.php' => '',
		],
		'assorted' => [
				'.', => '',
				'..' => '',
				'somefile.php' => '',
				'another-file.php' => '',
				'scriptfile.js' => '',
				'xmlfile.xml' => '',
				'jsonfile.json' => '',
				'count-thisfile-please.php' => '',
				'donotcountthisfile.phpx' => '',
				'yet-anothercountable.file.php' => '',
				'whatkindoffileisthis.php.crazy' => '',
		],
		'empty' => [],
	],
],
```

Using this strategy, the test data focuses on what is expected when invoking the function.